main.cpp

```diff
@@ -101,52 +135,116 @@ std::string Piano::displayChoosenSound(double soundKnobValue)
     return organSound;
 }
 
+double Piano::audio(int notePressed, double volume, bool isFootPedalPressed)
+{
+    if (isFootPedalPressed)
+    {
+        return 440.0 * volume;
+    }
+    return notePressed;
+}
+
 
 struct Microwave
 {
-    bool doorIsOpen = true;
-    double secondsRemaining = 0.0;
-    double power = 700.0;
-    int presetDishInput = 0;
+    Microwave();
+    bool doorIsOpen;
+    double secondsRemaining, power;
+    int presetDishInput;
 
     struct Preset
     {
-        float weightInGrams = 500.0f; 
-        int seconds = 1200;  
-        int power = 900; 
-        std::string presetName = "Potato";
+        float weightInGrams; 
+        int seconds;  
+        int power; 
+
+        Preset(float WeightInGrams,int Seconds,int Power);
     };
 
     void cookPotato( Preset potato );
     void cookByWeight(double selectedWeight, double power);
 };
 
+Microwave::Microwave()
+{
+    doorIsOpen = true;
+    secondsRemaining = 0.0;
+    power = 700.0;
+    presetDishInput = 0;
+    std::cout << "Instance of 'Microwave' has been constructed" << std::endl;
+}
+
+Microwave::Preset::Preset(float WeightInGrams,int Seconds,int Power)
+{
+    weightInGrams = WeightInGrams; 
+    seconds = Seconds;  
+    power = Power; 
+    std::cout << "Instance of 'Preset' has been constructed" << std::endl;
+}
+
 void Microwave::cookPotato(Microwave::Preset potato)
 {
     power = potato.power;
     secondsRemaining = potato.seconds;
+    std::cout << "Potato has been cooked" << std::endl;
 }
 
 void Microwave::cookByWeight(double selectedWeight, double selectedPower)
 {
     power = selectedPower;
     secondsRemaining = selectedWeight / selectedPower;
-}
+    std::cout << "Cooking at " << power << "w power for " << secondsRemaining << "seconds" << std::endl;
 
+}
 
 struct Airport
 {
-    int numPlanes = 50;
-    int numPassengers = 1000;
-    int numCafes = 10;
-    int coffeeStock = 500;
-    int averageMinutesInQueue = 120;
+    Airport();
+    int numPlanes, numPassengers, numCafes, coffeeStock, averageMinutesInQueue;
+
+    struct Aeroplane
+    {
+        int numSeats; 
+        float fuelLevel;  
+
+        Aeroplane(int setNumSeats, float setFuelLevel);
+    };
 
-    void boardPassengers(bool planeIsReady, int numPassengers);
+    void boardPassengers(bool planeIsReady, Airport::Aeroplane aeroplane);
     bool coffeeRestockNeeded(int coffeeSales);
-    bool passengersCanEnter(int numPassengers, int averageMinutesInQueue);
+    bool passengersCanEnter(int setNumPassengers, int setAverageMinutesInQueue);
```

Comment:
change these variable names to not contain verbs
Name things what they are.
Name functions based on the action they perform
Variables represent things. 
functions represent actions.
- [ ] revise